src/index.css

```
@@ -490,6 +490,46 @@
+    --baseBorderHover: var(--ds-border-secondary);
+    --baseLine: var(--ds-border-tertiary);
+    --accentText: var(--ds-action-fg);
+    --accentTextContrast: var(--ds-text-primary);
+    --accentBg: var(--ds-action-bg);
+    --accentBgSubtle: var(--ds-action-bg);
+    --accentBgHover: var(--ds-action-bg-hover);
+    --accentBgActive: var(--ds-action-bg-hover);
+    --accentBorder: var(--ds-border-accent);
+    --accentSolid: var(--color-primary);
+    --accentSolidHover: var(--color-primary);
+    border: 0.5px solid var(--ds-border-tertiary);
```

Stylelint (17.12.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)